cfy_manager/components/postgresql_server/postgresql_server.py

```diff
@@ -483,17 +506,28 @@ def _configure_cluster(self):
         fd, tmp_path = mkstemp()
         os.close(fd)
         with open(tmp_path, 'w') as pg_conf:
-            pg_conf.write(
-                "log_destination = 'stderr'\nlogging_collector = on\n"
-                "log_directory = '{0}'\nlog_filename = 'postgresql-%a.log'\n"
-                "log_truncate_on_rotation = on\nlog_rotation_age = 1d\n"
-                "log_rotation_size = 0\nlog_line_prefix = '< %m >'\n"
-                "log_timezone = 'UCT'\ndatestyle = 'iso, mdy'\n"
-                "timezone = 'UCT'\nlc_messages = 'en_US.UTF-8'\n"
-                "lc_monetary = 'en_US.UTF-8'\nlc_numeric = 'en_US.UTF-8'\n"
-                "lc_time = 'en_US.UTF-8'\nshared_buffers = 128MB\n"
-                "default_text_search_config = 'pg_catalog.english'\n".format(
-                    POSTGRES_LOG_PATH))
+            pg_params = {
+                'log_destination': "'stderr'",
+                'logging_collector': "on",
+                'log_directory': "'{0}'".format(POSTGRES_LOG_PATH),
+                'log_filename': "'postgresql-%a.log'",
+                'log_truncate_on_rotation': "on",
+                'log_rotation_age': "1d",
+                'log_rotation_size': 0,
+                'log_line_prefix': "'< %m >'",
+                'log_timezone': "'UCT'",
+                'datestyle': "'iso, mdy'",
+                'timezone': "'UCT'",
+                'lc_messages': "'en_US.UTF-8'",
+                'lc_monetary': "'en_US.UTF-8'",
+                'lc_numeric': "'en_US.UTF-8'",
+                'lc_time': "'en_US.UTF-8'",
```

Comment:
i think all these should well work without the additional `'` quotes (as in, `x=y` is valid syntax in postgres.conf)

Reply:
We should also perhaps use all of these for non-cluster setup as well, so that we use mostly the same params on both.